Update serialization of timing functions

csnardi opened the pull request:
According to https://drafts.csswg.org/css-timing/#serializing-a-timing-function:

> The keyword values `ease`, `linear`, `ease-in`, `ease-out`, and `ease-in-out` are serialized as-is, that-is, they are not converted to the equivalent `cubic-bezier()` function.

Update two tests to match this behavior.

<!-- Reviewable:start -->

<!-- Reviewable:end -->

w3c-bots commented:
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/26709)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/360767080)

csnardi commented:
@ewilligers Can you review this?

csnardi committed:
Update serialization of timing functions

According to https://drafts.csswg.org/css-timing/#serializing-a-timing-function, "The keyword values ease, linear, ease-in, ease-out, and ease-in-out are serialized as-is, that-is, they are not converted to the equivalent cubic-bezier() function". Update two tests to match this behavior.

csnardi commented:
@ewilligers Could you also merge (or should I bug someone for permissions to do that)?
